<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CORInspectionDialog</class>
 <widget class="QDialog" name="CORInspectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1593</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COR Inspection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="mainLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="lessButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="currentButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="moreButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QVBoxLayout" name="imagePlotLayout"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="step">
       <property name="prefix">
        <string>step: </string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="plotToolbarLayout"/>
       </item>
       <item>
        <widget class="QLabel" name="instructionText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the best reconstruction.&lt;br/&gt;Selecting the left or right image centres the search space on their rotation centre.&lt;br/&gt;Step size is halved by selecting the centre image.&lt;br/&gt;Once sufficient quality is reached on the centre image click &amp;quot;Finish&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lessButton</tabstop>
  <tabstop>currentButton</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>step</tabstop>
  <tabstop>finishButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
